.github/workflows/build.yml

```diff
@@ -119,6 +119,55 @@ jobs:
       - name: Test
         run: cd build && ctest -C Release -j3 --output-on-failure
 
+  freebsd:
+    name: FreeBSD
+    runs-on: ubuntu-latest
+    if: github.event_name != "pull_request"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
why this version instead of "latest"?

Reply:
In a previous PR, I moved to using pinned dependency versions, with Dependabot updating them periodically. This is a more secure, locked-down use of third-party code.